docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -66,6 +66,23 @@ TODO
 
 TODO
 
+### `tensorrt-inference`
+
+#### Introduction
+
+Inference PyTorch/ONNX models on NVIDIA GPUs with [TensorRT](https://developer.nvidia.com/tensorrt).
+
+#### Metrics
+
+| Name                                      | Unit      | Description                                                                                              |
+|-------------------------------------------|-----------|----------------------------------------------------------------------------------------------------------|
+| tensorrt-inference/gpu_lat_ms_mean        | time (ms) | The mean GPU latency to execute the kernels for a query.                                                 |
+| tensorrt-inference/gpu_lat_ms_99          | time (ms) | The percentile 99 GPU latency to execute the kernels for a query.                                        |
```

Comment:
`The percentile 99` ->  `The 99th percentile`? Same to others.

Reply:
updated, thanks